templates/web/base/admin/problem_row.html

```diff
@@ -38,6 +38,22 @@
             [%- IF problem.is_closed %]<br>[% prettify_state('closed') %]: [% PROCESS format_time time=problem.lastupdate %][% END -%]
             [%- IF problem.is_open %]<br>[% loc('Last&nbsp;update:') %] [% PROCESS format_time time=problem.lastupdate %][% END -%]
         </small></td>
+        [%- IF c.user.has_permission_to('planned_reports', c.cobrand.body.id ) %] 
```

Comment:
You can see other users of has_permission_to in this same template check `problem.bodies_str_ids`, not `c.cobrand.body.id` - good to be consistent. (Also, you could theoretically be looking at a problem from a different body on one body and thus not have permission.)